src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1739,42 +1739,47 @@ private static SyntaxToken GetImplicitConstructorBodyToken(CSharpSyntaxNode cont
             }
         }
 
-        internal static BoundCall GenerateObjectConstructorInitializer(MethodSymbol constructor, DiagnosticBag diagnostics)
+        internal static BoundCall GenerateBaseConstructorInitializer(MethodSymbol constructor, DiagnosticBag diagnostics)
```

Comment:
>GenerateBaseConstructorInitializer [](start = 34, length = 34)

I think the old name was reflecting that this is a simplified form of a constructor initializer that calls parameter-less base constructor. The new name should reflect that as well, but it looks like it doesn't.